hms-app/src/components/input-forms/CategoryForm.tsx

```diff
@@ -87,10 +103,16 @@ export default function CategoryForm(props: IProps) {
           icon: <Check />,
           autoClose: 2000,
         })
+        onSuccess()
       }
     })
   }
 
+  function clearForm() {
+    setCategoryTitle('')
+    setCategoryDescription('')
```

Comment:
I think we could write

setCategory({
          title: '',
          description: '',
        })